src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,48 @@
+.nav {
+  &__list {
+    list-style: none;
+    display: flex;
+    align-items: center;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item {
+    position: relative;
+    margin-right: 20px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    display: inline-block;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 60px;
+    color: $color-main-text;
+    text-transform: uppercase;
+    text-decoration: none;
+
+    &.is-active {

```

Comment:
create separate active modifier for the links, and move these styles to those modifier
use is-active class only in html for tests passing